src/StreetNameRegistry.Projections.Integration/StreetNameLatestItem.cs

```diff
@@ -0,0 +1,91 @@
+        public string? NameEnglish { get; set; }
+
+        public string? HomonymAdditionDutch { get; set; }
+        public string? HomonymAdditionFrench { get; set; }
+        public string? HomonymAdditionGerman { get; set; }
+        public string? HomonymAdditionEnglish { get; set; }
+
+        public bool IsRemoved { get; set; }
+
+        public string PuriId { get; set; }
+        public string Namespace { get; set; }
+        public string VersionAsString { get; set; }
+        private DateTimeOffset VersionTimestampAsDateTimeOffset { get; set; }
+
+        public Instant VersionTimestamp
+        {
+            get => Instant.FromDateTimeOffset(VersionTimestampAsDateTimeOffset);
+            set
+            {
+                VersionTimestampAsDateTimeOffset = value.ToDateTimeOffset();
+                VersionAsString = new Rfc3339SerializableDateTimeOffset(value.ToBelgianDateTimeOffset()).ToString();
+            }
+        }
+
+        public StreetNameLatestItem()
+        {  }
+    }
+
+    public sealed class StreetNameLatestItemConfiguration : IEntityTypeConfiguration<StreetNameLatestItem>
+    {
+        internal const string TableName = "streetname_latest_items";
+
+        public void Configure(EntityTypeBuilder<StreetNameLatestItem> builder)
+        {
+            builder.ToTable(TableName, Schema.Integration)
+                .HasKey(x => x.PersistentLocalId)
+                .IsClustered();
+            builder.Property(x => x.PersistentLocalId).HasColumnName("persistent_local_id");
+
+            builder.Property(x => x.NisCode).HasColumnName("nis_code");
+            builder.Property(x => x.Status).HasColumnName("status");
+
+            builder.Property(x => x.NameDutch).HasColumnName("name_dutch");
+            builder.Property(x => x.NameFrench).HasColumnName("name_french");
+            builder.Property(x => x.NameGerman).HasColumnName("name_german");
+            builder.Property(x => x.NameEnglish).HasColumnName("name_english");
+
+            builder.Property(x => x.HomonymAdditionDutch).HasColumnName("homonym_addition_dutch");
+            builder.Property(x => x.HomonymAdditionFrench).HasColumnName("homonym_addition_french");
+            builder.Property(x => x.HomonymAdditionGerman).HasColumnName("homonym_addition_german");
+            builder.Property(x => x.HomonymAdditionEnglish).HasColumnName("homonym_addition_english");
+
+            builder.Property(x => x.IsRemoved).HasColumnName("is_removed");
+
+            builder.Property(x => x.PuriId).HasColumnName("puri_id");
+            builder.Property(x => x.Namespace).HasColumnName("namespace");
+            builder.Property(x => x.VersionAsString).HasColumnName("version_as_string");
+            builder.Property(StreetNameLatestItem.VersionTimestampBackingPropertyName).HasColumnName("version_timestamp");
+
+            builder.Ignore(x => x.VersionTimestamp);
+
+            builder.HasIndex(x => x.PersistentLocalId);
```

Comment:
Maybe add an index on NisCode? Will be important when filtering for verdachte gevallen API?